src/main/java/seedu/nutrigoals/model/user/Weight.java

```diff
@@ -0,0 +1,50 @@
+package seedu.nutrigoals.model.user;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.nutrigoals.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Weight object
+ */
+public class Weight {
+    public static final String VALIDATION_REGEX = "\\d+";
```

Comment:
Would it be better to replace this with `^(?:[3-9]\d{0,1}|[1,2]\d{0,2})$`? This matches 30 but not 300, and matches both 22 and 200. 🤔